docs/source/howto/workflows.rst

```diff
@@ -20,6 +20,172 @@ Running a predefined workflow
 
 `#3992`_
 
+The first step to running a predefined workflow is loading the work function or work chain class that defines the workflow you want to run.
+This can of course be achieved by importing the function or class from its respective module:
+
+.. code-block:: python
+
+    from aiida.workflows.arithmetic.add_multiply import add_and_multiply              # Import the add_and_multiply work function
+    from aiida.workflows.arithmetic.multiply_add import MultiplyAddWorkChain       # Import the MultiplyAddWorkChain WorkChain
+
+However, a more robust way of loading a workflow is using the ``WorkflowFactory``, for example:
```

Comment:
Reading this now, I think it would be better to just show what is best practice (factory) and mention that this is basically equivalent to an import, but more robust because of the plugin/entrypoint system. Starting the how-to with *what not to do* is more confusing than helpful and also runs counter the idea that a how-to should show the solution to the problem with only minimal context and additional explanations.

Reply:
I agree, starting with what not to do is probably not a great idea. I've removed the examples that imports the workflows, and reorganised the text.